### PR TITLE
[Ram] Modifies to add javadoc comments 

### PR DESCRIPTION
**Pull request by ramgsuri**
Patch for #2 , Modifies to add javadoc comments in Product Controller , Order Controller, ProductCategory Controller and all domain models

### CODE REVIEW

**Comment by anantjain6** [comment]
Patch for #2

**Comment by ramgsuri** [comment]
Hi @anantjain6 let me know if any review comments otherwise please close the PR.

**Comment by anantjain6** [comment]
@ramgsuri First of all thanks for your contribution

Sorry for the late response

I have added review comments on code changes

Also, I have following comments:-

1.  Comments are missing at "HomeController"
2.  Comments are missing at "viewMyOrder" method of "OrderController"
3.  Comments are missing at "OrderProduct" Model

Your suggestions are also welcome

**Comment by ramgsuri** [comment]
Review Comments Incorporated #19

**Review by anantjain6 — CHANGES_REQUESTED** [review]

File: src/main/java/me/anant/PMS/controller/OrderController.java [hunk 1]
```diff
@@ -97,29 +119,51 @@ public ModelAndView viewMyOrder(Principal principal) {
 		modelAndView.addObject("orderList", user.getOrders());
 		return modelAndView;
 	}
-	
+
+	/**
+	 * This GET api is responsible to List All Orders.
+	 * @return
```
[the remark this answers is not in the document]
Comment (ramgsuri) [reply src/main/java/me/anant/PMS/controller/OrderController.java]: Done

File: src/main/java/me/anant/PMS/controller/ProductController.java [hunk 2]
```diff
@@ -55,7 +70,13 @@ public ModelAndView list() {
 		modelAndView.addObject("pList", pList);
 		return modelAndView;
 	}
-	
+
+	/**
+	 * This GET api is responsible to delete the product.
+	 * @param id
+	 * @param redirectAttributes
+	 * @return
```
[the remark this answers is not in the document]
Comment (ramgsuri) [reply src/main/java/me/anant/PMS/controller/ProductController.java]: Done

**Review by anantjain6 — APPROVED** [review]

**Comment by ramgsuri** [comment]
hi @anantjain6 can you merge the PR and please close this PR.

**Comment by anantjain6** [comment]
@ramgsuri It solves the issue perfectly

Can you add a commit for deleting "EmailConfig" class?

Actually, the remote master branch has been updated due to which it is creating conflict. I can still push your codes by resolving conflict but in that way, I'll not able to maintain full credit of your work.

If you add commit for deleting EmailConfig class then your pull request can be merged smoothly with full credits of your work.

**Comment by ramgsuri** [comment]
Ohh ok Thanks a lot, thats the reason I was wondering, what's missing here 👍

**Comment by ramgsuri** [comment]
Hi @anantjain6 then this would also require delete in EmailService would that be ok as it is Autowirred in EmailService ?

**Comment by anantjain6** [comment]
@ramgsuri EmailService will get auto-merged. No action required for that

**Comment by ramgsuri** [comment]
Done

**Pull request merged**